ZenPacks/zenoss/Docker/zenpack.yaml

```diff
@@ -1,289 +1,294 @@
-                format: "%7.2lf%s"
-
-              Write:
-                dpName: blkio_io_write
-                format: "%7.2lf%s"
-
-          IO Throughput:
-            units: bytes/sec
-            miny: 0
-            base: true
-
-            graphpoints:
-              Total:
-                dpName: blkio_bytes_total
-                format: "%7.2lf%s"
-
-              Read:
-                dpName: blkio_bytes_read
-                format: "%7.2lf%s"
-
-              Write:
-                dpName: blkio_bytes_write
-                format: "%7.2lf%s"
-
-      DockerContainer-Status:
-        description: "Docker container status monitoring using \"docker ps\" via SSH."
-        targetPythonClass: ZenPacks.zenoss.Docker.DockerContainer
-
-        datasources:
-          status:
-            type: COMMAND
-            usessh: true
-            commandTemplate: '/usr/bin/env sudo docker ps -a --no-trunc 2>/dev/null'
-            parser: ZenPacks.zenoss.Docker.parsers.ps
-            component: ${here/id}
-            eventClass: /Ignore
-            severity: Clear
-            cycletime: 60
-
-      DockerContainer-Size:
-        description: "Docker container size monitoring using \"docker ps -s\" via SSH."
-        targetPythonClass: ZenPacks.zenoss.Docker.DockerContainer
-
-        datasources:
-          size:
-            type: COMMAND
-            usessh: true
-            commandTemplate: '/usr/bin/env sudo docker ps -a -s --no-trunc 2>/dev/null'
-            parser: ZenPacks.zenoss.Docker.parsers.ps
-            component: ${here/id}
-            eventClass: /Ignore
-            severity: Clear
-            cycletime: 600
-
-            datapoints:
-              size: GAUGE_MIN_0
-              size_virtual: GAUGE_MIN_0
-
-        graphs:
-          Storage Usage:
-            units: bytes
-            miny: 0
-            base: true
-
-            graphpoints:
-              Size:
-                dpName: size_size
-                format: "%7.2lf%s"
-
-              Virtual Size:
-                dpName: size_size_virtual
-                format: "%7.2lf%s"
-
-
-classes:
-  DockerContainer:
-    base: [zenpacklib.Component]
-    label: Docker Container
-    order: 1
-
-    properties:
-      image:
-        label: Image
-        order: 4.01
-
-      command:
-        label: Command
-        order: 4.02
-
-      created:
-        label: Created
-        order: 4.03
-
-      ports:
-        label: Ports
-        order: 4.04
-
-    monitoring_templates:
-      - DockerContainer
-      - DockerContainer-Status
-      - DockerContainer-Stats
-      - DockerContainer-Size
-
-    dynamicview_group: Docker Containers
-    dynamicview_views: [service_view]
-    dynamicview_weight: 35
-    dynamicview_relations:
-      impacted_by:
-        - docker_host
-
-class_relationships:
-  - Products.ZenModel.Device.Device(docker_containers) 1:MC (docker_host)DockerContainer
+name: ZenPacks.zenoss.Docker

```

Comment:
I can't see why GitHub thinks the entire zenpack.yaml file has been changed, but you should sort this out. We don't want to unnecessarily complicated the git history for this file.